Source/CMakeLists.txt

```diff
@@ -35,7 +35,15 @@ if (USE_SKIA)
 endif ()
 
 if (USE_SYSPROF_CAPTURE)
```

Comment:
Unimportant: you could alternatively write:

```
if (USE_SYSTEM_SYSPROF_CAPTURE)
    # ...
elseif (USE_SYSPROF_CAPTURE)
    # ...
endif ()
```

and then you have one fewer level of indentation, if you prefer.

Reply:
Doing this would require also setting a dependency between both options, which is probably a good thing to add anyway to ensure that `USE_SYSTEM_SYSPROF_CAPTURE` will always be disabled when `USE_SYSPROF_CAPTURE` is disabled as well.